conda_env/specs/__init__.py

```diff
@@ -0,0 +1,4 @@
+from binstar import BinstarSpec
```

Comment:
FYI, this will get you in trouble.  It's a implied relative import that breaks in Python 3.  Though I'm setting a bad example in this code base, it's always a good idea to include `from __future__ import absolute_import`.  This needs to be `from .binstar import BinstarSpec`.